launchable/commands/helper.py

```diff
@@ -57,3 +59,19 @@ def find_or_create_session(
                 is_observation=is_observation,
             )
             return read_session(saved_build_name)
+
+
+def _check_observation_mode_status(session: str, is_observation: bool):
+    if is_observation is False:
+        return
+
+    client = LaunchableClient()
+    res = client.request("get", session)
+
+    # only check when the status code is 200 not to stop the command
+    if res.status_code == 200:
+        is_observation_in_recorded_session = res.json().get("isObservation", False)
+        if is_observation and is_observation_in_recorded_session is False:
```

Comment:
[must]
Better and rather common to write like `if is_observation and not is_observation_in_recorded_session`. 
`is` in Python is used to compare two variables pointing to the same object in memory. I know it works for comparing boolean, since `True` and `False` are singleton in Python, but it is Pythonic to just write like `if a` to evaluate `a==True` or `a==False`.